apps/angular/src/core/models/anime.ts

```diff
@@ -0,0 +1,47 @@
+import { AnimeStatus } from './anime-status';
+import { AnimeType } from './anime-type';
+
+import { DateTimeRange } from './date-time-range';
+
+/** Represents anime. */
+export type Anime = {
+
+	/** Id. */
+	readonly id: number;
+
+	/** Created. */
+	readonly created: Date;
+
+	/** Modified. */
+	readonly modified: Date;
+
+	/** Anime title in English. */
+	readonly titleEng: string;
+
+	/** Anime title in Japanese. */
+	readonly titleJpn: string;
+
+	/** Image - anime poster. */
+	readonly image: string;
```

Comment:
If this image is a link we can give a hint about it using property name. Besides, we can be more descriptive and use `poster` instead of generic 'image':
```suggestion
	/** Anime poster URL. */
	readonly posterUrl: string;
```
This is an example, you can use alternative naming if you wish.